time_tracker_api/projects/projects_model.py

```diff
@@ -65,24 +65,22 @@ def __init__(self):
             partition_key_attribute='tenant_id',
             mapper=ProjectCosmosDBModel,
         )
-            
-    def find_all_with_customer_id_in_list(
+
+    def find_all_v2(
         self,
         event_context: EventContext,
-        customer_ids_list: List[str],
+        ids_list: List[str],
+        customers_ids_list: List[str] = None,
```

Comment:
I think that customer_ids is better, customers_ids_list is too long .

Reply:
Thank you for your observation Kelly